docs/smtchecker.rst

```diff
@@ -39,8 +39,16 @@ The potential warnings that the SMTChecker reports are:
 - ``<failing  property> happens here.``. This means that the SMTChecker proved that a certain property fails. A counterexample may be given, however in complex situations it may also not show a counterexample. This result may also be a false positive in certain cases, when the SMT encoding adds abstractions for Solidity code that is either hard or impossible to express.
 - ``<failing property> might happen here``. This means that the solver could not prove either case within the given timeout. Since the result is unknown, the SMTChecker reports the potential failure for soundness. This may be solved by increasing the query timeout, but the problem might also simply be too hard for the engine to solve.
 
-It is currently an experimental feature, therefore in order to use it you need
-to enable it via :ref:`a pragma directive<smt_checker>`.
+To enable the SMTChecker, you must select :ref:`which engine should run<smtchecker_engines>`,
+where the default is no engine. Selecting the engine enables the SMTChecker on all files,
+whereas the previous behaviour was to only analyze contracts that had ``pragma experimental SMTChecker;``.
+
+.. note::
+
+  Prior to Solidity 0.8.4, the default way to enable the SMTChecker was via
+  ``pragma experimental SMTChecker;``. That pragma has been deprecated, and
+  although it still enables the SMTChecker for backwards compatibility, it will
+  be removed in Solidity 0.9.0.
```

Comment:
After your clarification that the [pragma is now supposed to change behavior to run on all files](https://github.com/ethereum/solidity/pull/11202#discussion_r609699846), I would mention that too:

```suggestion
To enable the SMTChecker, you must select :ref:`which engine should run<smtchecker_engines>`,
where the default is no engine. Selecting the engine enables the SMTChecker on all files.

.. note::

  Prior to Solidity 0.8.4, the default way to enable the SMTChecker was via
  ``pragma experimental SMTChecker;`` and only the contracts containing the pragma
  would be analyzed. That pragma has been deprecated, and although it still enables
  the SMTChecker for backwards compatibility, it will be removed in Solidity 0.9.0.
  Note also that now using the pragma even in a single file enables the SMTChecker for all files.
```